src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -266,10 +265,6 @@ const DataSubview: FunctionComponent<Props> = ({
 
   const numCheckedSamples = checkedSamples?.length;
   const hasCheckedSamples = numCheckedSamples > 0;
-  const hasTooManyCheckedSamples = numCheckedSamples > 2000;
```

Comment:
Is there any checking for too many samples anywhere anymore? I just searched the whole FE code base and I can't find `2000` show up anywhere other than in English ("Depending on the Tree Type, add up to 2000 samples", `src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/index.tsx`). Are we actually enforcing this in code for the FE still?

If not, I wonder what happens if someone submits a million gisaid samples. Heck, even if we did enforce it in FE, I wonder what would happen if someone just manually hit the backend API with a million gisaid samples. Probably not something to worry about, though.

Reply:
oh crap, very good catch. I definitely don't want to disabled that upper limit